share/picongpu/dockerfiles/README.rst

```diff
@@ -0,0 +1,62 @@
+Dockerfiles
+===========
+
+User
+----
+
+Download a docker container and execute it locally.
+Opens a bash prompt with a fully set up PIConGPU environment.
+
+.. code:: bash
+
+    nvidia-docker pull ax3l/picongpu
+    nvidia-docker run -it ax3l/picongpu
+
+or in singularity:
+
+.. code:: bash
+
+    # singularity pull ax3l/picongpu
+    singularity run --nv picongpu.img
+
+Alternatively, run an already configured and fully build example.
+This exposes the ISAAC port to connect via the webclient to.
+
+.. code:: bash
+
+    nivida-docker pull ax3l/picongpu
+    nvidia-docker run -p 2459:2459 -t ax3l/picongpu:0.3.0 /bin/bash -lc start_lwfa
+    # open firefox and isaac client
+
+Maintainer / Developer
+----------------------
+
+Build a new container image (build all dependencies).
+First, set ``PICONGPU_BRANCH`` inside the ``Dockerfile`` accordingly to choose a version.
+Set the same version of PIConGPU inside the ``Singularity`` bootstrap recipe.
+You can also push the result to dockerhub and singularity-hub (you need an account there).
+
+.. code:: bash
+
+    cd ubuntu-1604
+
+    # docker image
+    docker build -t ax3l/picongpu:0.3.0 .
```

Comment:
note to myself: for building 0.3.0, I added the `0001gpus_isaac.cfg` to this dir first and added
```Docker
COPY       0001gpus_isaac.cfg $HOME/paramSets/lwfa/submit/
```